### PR TITLE
Add new rules from Stylelint 8

### PR DESCRIPTION
**Pull request by efedorenko**
Guys, I reviewed a list of new rules in Stylelint 8 once again and added a few new ones to our config.

Any objections?

### CODE REVIEW

**Comment by efedorenko** [comment]
BTW, I tried [stylelint-validator](https://github.com/csstree/stylelint-validator/) as a replacement for some of the deprecated rules, but they still can't parse most of the Sass :(

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stylelint-config-wildbit",
-  "version": "1.2.0",
+  "version": "8.1.1",
```
Comment (matt-west) [inline package.json:3]: Should this reflect the version of our module, or mimic the `stylelint` version?

**Review by matt-west — APPROVED** [review]
Look good. Thanks @efedorenko!

**Review by derekrushforth — APPROVED** [review]
Looks good, no objections here. Thanks for going through this.

File: package.json (hunk 1 above)
Comment (derekrushforth) [reply package.json:3]: That's a big bump! :)

Comment (efedorenko) [reply package.json:3]: I fugured that it’s going to be much easier to use correct version of config if we match Stylelint version. Maybe at least for the first two digits of semver. Any objections?

Comment (matt-west) [reply package.json:3]: Okay, that works for me. :)

Comment (derekrushforth) [reply package.json:3]: Sounds good!

**Pull request merged**